lib/routes/account.js

```diff
@@ -887,7 +896,19 @@ module.exports = function (
         log.begin('Account.deviceDestroy', request)
         var sessionToken = request.auth.credentials
         var uid = sessionToken.uid
-        db.deleteDevice(uid, request.payload.id).then(reply, reply)
+        var id = request.payload.id
+        db.deleteDevice(uid, id).then(
+          function (result) {
+            log.event('device', request, {
+              action: 'delete',
```

Comment:
IIUC, you and @dannycoates both expressed a preference for namespaced events here, I would support either "device:delete" or "device.delete" rather than adding hierarchy via items in the event body.